pkg/spoke/spokeagent.go

```diff
@@ -26,6 +26,13 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client/apiutil"
 )
 
+const (
+	// TODO expose a flag to set the worker for each controller
+	appliedManifestWorkFinalizeControllerWorkers = 10
```

Comment:
can we add a comment on why it needs to be 10?

Reply:
added, PTAL